src/control/cmd/dmg/pretty/pool_test.go

```diff
@@ -209,6 +209,310 @@ Rebuild failed, rc=0, status=2
 	}
 }
 
+func TestPretty_PrintPoolQueryTargetResp(t *testing.T) {
+	for name, tc := range map[string]struct {
+		pqtr        *control.PoolQueryTargetResp
+		expPrintStr string
+	}{
+		"empty response": {
+			pqtr: &control.PoolQueryTargetResp{},
+			expPrintStr: `
+`,
```

Comment:
In these cases would be slightly more readable to use "\n".

Reply:
done